modules/oc-adm-by-example-content.adoc

```diff
@@ -785,3 +929,36 @@ Verify the image identity contained in the image signature
 ----
 
 
+
+== oc adm wait-for-node-reboot
+Wait for nodes to reboot after running `oc adm reboot-machine-config-pool`
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Wait for all nodes to complete a requested reboot from 'oc adm reboot-machine-config-pool mcp/worker mcp/master'
+  oc adm wait-for-node-reboot nodes --all
+  
+  # Wait for masters to complete a requested reboot from 'oc adm reboot-machine-config-pool mcp/master'
+  oc adm wait-for-node-reboot nodes -l node-role.kubernetes.io/master
+  
+  # Wait for masters to complete a specific reboot
+  oc adm wait-for-node-reboot nodes -l node-role.kubernetes.io/master --reboot-number=4
+----
+
+
+
+== oc adm wait-for-stable-cluster
+wait for the platform operators to become stable
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Wait for all clusteroperators to become stable
+  oc adm clusteroperator wait-for-stable-cluster
```

Comment:
same , wrong command here :
oc adm clusteroperator wait-for-stable-cluster
error: unknown command "clusteroperator wait-for-stable-cluster"
See 'oc adm -h' for help and examples